examples/gallery/images/rgb_image.py

```diff
@@ -0,0 +1,39 @@
+"""
+RGB Image
+---------
+The :meth:`pygmt.Figure.grdimage` method can be used to plot Red, Green, Blue
+(RGB) images, or any 3-band false color combination. Here, we'll use
+:meth:`rioxarray.open_rasterio` to read a GeoTIFF file into an
```

Comment:
Is it possible to make this work as a link to https://corteva.github.io/rioxarray/stable/rioxarray.html#rioxarray.open_rasterio?

If this is not easy to do, then we leave it as is. At least in the code example below, the link works.

Reply:
Ah yes, good catch. I used the wrong intersphinx directive, should be `:py:func:`
```suggestion
:py:func:`rioxarray.open_rasterio` to read a GeoTIFF file into an
```

Reply:
Ah, great. Thanks!